make.ps1

```diff
@@ -372,7 +374,16 @@ if ($command -eq "all" -or $command -eq "clean")
 		Rename-Item $extractedDir.Name (Split-Path -leaf $env:ENGINE_DIRECTORY)
 
 		rm $env:AUTOMATIC_ENGINE_EXTRACT_DIRECTORY -r
+	}
+}
+
 
+
+# Run the same command on the engine's make file
+if ($command -eq "all" -or $command -eq "clean" -or $command -eq "check")
```

Comment:
This doesn't look correct. If the engine exists and we take at line 319 we'll run the command twice, won't we?

Reply:
I honestly don't remember even making this change, so I can't explain my motivation for doing this.

Why are we only fetching the engine only on all or clean to begin with? Surely we should *always* be checking for and fetching the engine, and then the individual `*-Command`s should be forcing a recompile if required, like the main makefile does.

Reply:
Updated to remove this change.

I think this was trying to work around the error messages from `rm` when you try to clean an already clean build. I have instead changed the SDK make.ps1 to use `Remove-Item -ErrorAction Ignore`, and will file a separate PR to make the same change in the engine make.ps1 to resolve the issue here when we update to the next playtest tag.